src/timerfd.rs

```diff
@@ -233,6 +246,7 @@ mod tests {
 
         assert_eq!(count, 1);
         assert!(now.elapsed() >= dur);
+        tfd.clear().expect("unable to clear the timer");
```

Comment:
I don't think that calling the clear function in this case is enough. You should also check that the `clear` function had the expected effect.

Reply:
Fixed.